satellite/modules/vm_manage.py

```diff
@@ -109,13 +176,27 @@ def vm_details(compute_name, compute_ip, vm_id):
 
 
 def get_packages(compute_ip, vm_ip, root_passwd):
+    """Get packages installed in virtual machine
+
+    :param compute_ip: IP address of compute on which vm is running
+    :param vm_ip: IP address of virtual machine
+    :param root_passwd: Root password of virtual machine
+
+    :return package_info: List of all packages in virtual machine
+    """
     vm_ip = vm_ip.split("/")[0]
     package_info = os.popen("ssh root@" + compute_ip + " 'sshpass -p " + root_passwd + " ssh -o StrictHostKeyChecking=no root@" + vm_ip + " rpm -qa'").readlines()
     package_info = package_info[2:]
     return package_info
 
 
 def get_repo(activation_name):
+    """Get repo list included in Activation name
+
+    :param activation_name: Name of activation
+
+    :return repo: dictionary of repo name and repo URL included in that activation
```

Comment:
if you start writing the first character in uppercase follow that everywhere.
**d**ictionary is in lowercase